api_tests/src/image.spec.ts

```diff
@@ -194,15 +202,19 @@ test("No image proxying if setting is disabled", async () => {
   ).toBeTruthy();
   expect(post.post_view.post.body).toBe("![](http://example.com/image2.png)");
 
-  let gammaPost = await resolvePost(delta, post.post_view.post);
-  expect(gammaPost.post).toBeDefined();
+  let betaPost = await waitForPost(
+    beta,
+    post.post_view.post,
+    res => res?.post.alt_text != null,
+  );
+  expect(betaPost.post).toBeDefined();
```

Comment:
Found the problem, gamma was fetching the post here before metadata was generated. This means the post url was federated as `Link` which doesnt include alt text. Have to subscribe to the community and wait for it to send the post out after metadata is generated. Also had to change from delta to beta because follow didnt go through from delta for some reason.